community/modules/scheduler/schedmd-slurm-gcp-v5-controller/versions.tf

```diff
@@ -0,0 +1,19 @@
+/**
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+*/
+
+terraform {
+  required_version = ">= 0.14.0"
```

Comment:
- Did you pick 0.14 for a reason?

I'm not sure whether we should be adding a provider and/or provider_meta block here given that there are 0 resources being directly managed in this module.

Reply:
@nick-stroud I appreciate your input on this question. I think the "right" thing to do is not to have any `provider` block whatsoever. In fact, I think that adding it would cause TFLint to fail. What I don't know one way/other is if one can add `provider_meta` in the absence of `provider`. I bet TFLint wouldn't care but Terraform itself might.

Reply:
provider_meta is only needed when a google (or google beta) provider is called out, and those are only needed when there is a resource or data block using the provider, so not here. 

I am not sure if this file is still needed/ if the `required_version` is needed if everything is a submodule. You might still want the `required_version` called out if there are features being used in the locals block that are not available before a certain point. 

Summary, `required_providers ` and `provider_meta` not needed. Beyond that if tflint doesn't complain, then I won't.

Reply:
This module is so simple I think we should probably lower the version to 0.13 but leave `required_version` in. (`for_each` and `count` for modules was introduced in 0.13 even if we somehow end up using them)

Reply:
I removed it and all the tests still passed. Fixed in the last pushed commit.